### DISCUSSION TITLE
Skiovox Extenions no loading

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#1022] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When i try downloading an extension in skiovox it says "Download connected interrupted can't download extension"

**Comment by CGRmags** [comment]
I have tried skebstore bwt adn I have been able to install extensions up until right now

**Reply by CGRmags** [reply]
@AshtonDavies help?

**Reply by v4pl** [reply, marked as answer]
@CGRmags 
try a new network?
maybe u did CAUB wrong?

**Reply by CGRmags** [reply]
wydm by Caub wrong? I tried all the networks

**Reply by S-PScripts** [reply]
> @CGRmags try a new network? maybe u did CAUB wrong?

marking as answer since OP inactive on this thread
